core/src/services/webhdfs/backend.rs

```diff
@@ -104,6 +106,17 @@ impl WebhdfsBuilder {
         }
         self
     }
+
+    /// Disable batch listing
+    ///
+    /// # Note
+    ///
+    /// When listing a directory, the backend will default to use batch listing.
+    /// If disable, the backend will list all files/directories in one request.
+    pub fn disable_list_batch(&mut self, disable: bool) -> &mut Self {
```

Comment:
OpenDAL's boolean config's API must be `disable_list_batch(&mut self)` instead accept a `disable`

Reply:
Fixed, Sorry, it was my oversight